solidity/contracts/converter/BancorFormula.sol

```diff
@@ -12,41 +13,29 @@ contract BancorFormula is IBancorFormula {
     uint8 private constant MIN_PRECISION = 32;
     uint8 private constant MAX_PRECISION = 127;
 
-    /**
-      * Auto-generated via 'PrintIntScalingFactors.py'
-    */
+    // Auto-generated via 'PrintIntScalingFactors.py'

```

Comment:
You probably need to change the `PrintIntScalingFactors.py` accordingly?

Reply:
I believe you refer to the FormulaSolidityPort - I might but not sure about the process there so need @barakman to look.